src/OpenTelemetry.AutoInstrumentation/Configurations/ConfigurationKeys.cs

```diff
@@ -108,27 +108,69 @@ public static class Traces
         /// </summary>
         public static class InstrumentationOptions
         {
+#if NETFRAMEWORK
             /// <summary>
-            /// Configuration key for GraphQL instrumentation to enable passing query as a document attribute.
+            /// Configuration key for ASP.NET instrumentation to enable capturing HTTP request headers as span tags.
             /// </summary>
-            public const string GraphQLSetDocument = "OTEL_DOTNET_AUTO_GRAPHQL_SET_DOCUMENT";
+            public const string AspNetInstrumentationCaptureRequestHeaders = "OTEL_DOTNET_AUTO_TRACES_ASPNET_INSTRUMENTATION_CAPTURE_REQUEST_HEADERS";
```

Comment:
Do we really need to separate aspnet and aspnet core?

Reply:
It was the main question I have raised on SIG meeting Other parts of the PR are more or less boring.

For this version - we have separate settings for SqlDatabaseStatements. And it was the reason I started with this approach.

Unification, less things to maintain/configure.

Reply:
For sure this approach is safer as if only one integration is requested to extract headers. If it's the same header name for eg asp.net and grpc and ones value is important not to be captured.
But from the other hand adds a lot of configuration overhead.

Reply:
Please check a15789f722179866b935d6d1dcb6acb225f34d10

Reply:
@rajkumar-rangaraj, @zacharycmontoya, @nrcventura, could you please check (whole PR, but env. vars, are most important IMO)?

Reply:
It looks like we have a precedent with some of our other environment variables for separating aspnet and aspnetcore, so I think that keeping the env vars separate here also make sense. I think separating the aspnet* and *client variables make sense because I can see different use cases for those. The client libraries are also different enough that I think it makes sense to keep their settings separated.